src/Symfony/Bundle/SecurityBundle/Resources/config/security_listeners.xml

```diff
@@ -258,6 +258,15 @@
             <argument type="service" id="event_dispatcher" on-invalid="null"/>
         </service>
 
+        <service id="security.authentication.session_expiration_listener" class="Symfony\Component\Security\Http\Firewall\SessionExpirationListener" public="false" abstract="true">
```

Comment:
You now also need to update the required version of the `symfony/security-http` component in the SecurityBundle.